Makefile

```diff
@@ -13,7 +13,7 @@ install:
 test: unit-test integration-test
 
 fixture-base = test/fixtures
-fixture-repos = next-article n-ui next-myft-email
+fixture-repos = next-article next-myft-email n-tracking # app, lambda, component
```

Comment:
Are these now used directly (i.e. not via `n-ui`)?

Reply:
The integration tests for this component are done by cloning these repos listed here and running tasks on them.